success-criteria.md

```diff
@@ -265,15 +248,7 @@ societies." Among the rationales supporting decentralization are choice and the
 freedom of information. These rationales help keep the market innovative,
 competitive and open to new entrants.
 
-The common element among these rationales is the accessibility to a wide array
-of diverse publishers. As U.S. Supreme Court Justice Brandeis
-[wrote](http://www.columbia.edu/itc/journalism/j6075/edit/readings/brandeis_concurring1.html):
-"Among free men, the deterrents ordinarily to be applied to prevent crime are
-education and punishment for violations of the law, not abridgment of the rights
-of free speech and assembly." [BRANDEIS] Thus, to exercise this freedom, people
-should have digital access to publishers, which equates to both the right to
-assembly and freedom of speech. Safeguarding and improving this accessibility
-and choice should be goals of the Business Group.
+The goals of this Business Group should be to understand how and why the interactions of users, publishers, ad technologies and marketers counteract or conflict with the goals of privacy-preserving users and regulations. Then to find and propose measures by which systems of publishers, technologies and marketers can be adjusted to continue to create profitable operations that supports a variety of freedoms while not counteracting a human right to privacy.
```

Comment:
I find the use of equates to be an issue in the original text. It is an aspect of those freedoms, but it is not the only expression of such freedoms and it focuses on the surface at the level of publishers, when much of the problems of digital access and privacy arise from the interactions that occur below the level of the publisher page and within the large advertising technology economy.

I would strike this entire containing paragraph and pull scope down to a clearer and more focused statement. I think that it would be more accurate to describe our goals as providing mechanisms and considerations for users to control their privacy while still interacting with ad systems that provide monetization.